### ISSUE TITLE
opal and sardonyx image builds failing during 'dpkg-reconfigure locales'

### ISSUE THREAD

**Issue by edmorley** [travis-ci/packer-templates#501]
I triggered opal/sardonyx builds (which failed) when testing another PR since I thought they were passing when #496 was merged - but the builds are actually failing on master.

For example:
```
    googlecompute: STDERR: debconf: unable to initialize frontend: Dialog
    googlecompute: debconf: (Dialog frontend will not work on a dumb terminal, an emacs shell buffer, or without a controlling terminal.)
    googlecompute: debconf: falling back to frontend: Readline
    googlecompute: debconf: unable to initialize frontend: Readline
    googlecompute: debconf: (This frontend requires a controlling tty.)
    googlecompute: debconf: falling back to frontend: Teletype
    googlecompute: Use of uninitialized value $_[1] in join or string at /usr/share/perl5/Debconf/DbDriver/Stack.pm line 111.
    googlecompute: Generating locales (this might take a while)...
    googlecompute: Generation complete.
    googlecompute: *** update-locale: Error: invalid locale settings:  LANG=en_US.UTF-8 LC_ALL="en_US.UTF-8" LANGUAGE="en_US.UTF-8"
```
(https://travis-ci.org/travis-infrastructure/packer-build/jobs/269421413#L1655)

See:
https://travis-ci.org/travis-infrastructure/packer-build/builds/269421412
https://travis-ci.org/travis-infrastructure/packer-build/builds/269421397

**Comment by spurti-chopra** [comment]
Facing same issue while trying to build Stevonie on Openstack Setup using Ubuntu 16.04 LE image via packer template. Although execution of recipe travis_build_environment::locale runs smoothly when ran directly without packer.

**Comment by spurti-chopra** [comment]
@meatballhat and @edmorley , I was able to get this issue resolved while doing test on Openstack setup for ubuntu xenial 16.04 by adding explicit settings in recipe travis_build_environemnt::locale

> execute 'dpkg-reconfigure locales' do
>   action :nothing
>     environment(
>     'DEBIAN_FRONTEND' => 'noninteractive'
>   )
> end

Although this setting (DEBIAN_FRONTEND' => 'noninteractive') is used across multiple packer-scripts , but is not getting used while running this particular recipe. Please let me know if explicit addition in recipe could help. I can raise a PR for this.

**Comment by edmorley** [comment]
This was fixed by travis-ci/travis-cookbooks#906 (albeit opal/sardonyx are now failing for other reasons).
